### PR TITLE
Gfieni2022 patch 1

### PR DESCRIPTION
**Pull request by gfieni2022**
Grace Fieni created a review for 18-847.

# Summary of Changes

- Please indicate the classes you are modifying:
  - 18-847

# Checklist:

- [x] I have added my entries to the `README.md` table of contents in the root directory as links.

Make sure that your contributions:

- [x] Are accurate and will not mislead the reader
- [x] Do not encourage or cause academic integrity violations
- [x] Do not offend professors or unfairly criticize the university

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,7 @@ expect from the core classes from the ECE and CS programs at CMU.
 - [24-104: Maker Series: Intro to Modern Making](electives/24104.md)
 - [80-180: Nature of Language](electives/80180.md)
 - [80-405: Game Theory](electives/80405.md)
+- [18-847: Data Center Computing](electives/18847.md)
```
Comment (mikinty) [inline README.md]: This should go before 18-847F

File: electives/18847.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# 18-847C: Data Center Computing
+
+| Category | Difficulty |
+|:-:       | :-:        |
+| Reviews  | 5          |
+| Project  | 6          |
+| Present  | 7          |
+
+This class gives a introduction to data center computing issues by reading recently published research papers addressing current problems that datacenters face today, as well as through a semester long research project into a data center problem.
```
Comment (mikinty) [inline electives/18847.md]:
```suggestion
This class gives an introduction to data center computing issues through reading research papers that address  problems that datacenters face today, as well as working through a semester long research project about a data center problem. 
```

File: electives/18847.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+|:-:       | :-:        |
+| Reviews  | 5          |
+| Project  | 6          |
+| Present  | 7          |
+
+This class gives a introduction to data center computing issues by reading recently published research papers addressing current problems that datacenters face today, as well as through a semester long research project into a data center problem. 
+
+## Paper Reviews
+
+Each class (meet twice a week), you are required to read and review two reseach papers. During class, these papers are presented and discussed. Sometimes, one of the reseachers who wrote the paper will visit for discussion. The most important part of the reviews is the critical analysis of where they could have improved and your discussion questions.
```
Comment (mikinty) [inline electives/18847.md]: Do you need to prepare your discussion questions beforehand? Are you graded on how much you participate in these discussions?

```suggestion
Each class (meets twice a week), you are required to read and review two research papers. During class, these papers are presented and discussed. Sometimes, one of the researchers who wrote the paper will visit for discussion. The most important part of the reviews is the critical analysis of where they could have improved and your discussion questions. 
```

File: electives/18847.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+This class gives a introduction to data center computing issues by reading recently published research papers addressing current problems that datacenters face today, as well as through a semester long research project into a data center problem. 
+
+## Paper Reviews
+
+Each class (meet twice a week), you are required to read and review two reseach papers. During class, these papers are presented and discussed. Sometimes, one of the reseachers who wrote the paper will visit for discussion. The most important part of the reviews is the critical analysis of where they could have improved and your discussion questions. 
+
+## Paper Presentations
+
+You are required to prepare 2-3 paper presentations throughout the semester. These presentations are 10-15 minutes long and can be relatively informal. Each class, two of these presentations occur, each one focusing on one of the papers for the class. These often take long to prepare, contributing to a higher number of hours spent on the class for those weeks.
```
Comment (mikinty) [inline electives/18847.md]: Why do these presentations take a long time to prepare? What are ways to help people get unstuck while they are making these presentations?

File: electives/18847.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+## Cons of the Class
+
+- Fast paced, difficult to ingest new information so frequently.
+- Not enough time is given to the project in class.
+- Reviews often take up much time in the week. 
+
+
+## Resources
+
+- [Github for the class] https://github.com/STAR-Research/18-847C-S22
```
Comment (mikinty) [inline electives/18847.md]:
```suggestion
- [Github for the class](https://github.com/STAR-Research/18-847C-S22)
```

File: electives/18847.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+## Paper Presentations
+
+You are required to prepare 2-3 paper presentations throughout the semester. These presentations are 10-15 minutes long and can be relatively informal. Each class, two of these presentations occur, each one focusing on one of the papers for the class. These often take long to prepare, contributing to a higher number of hours spent on the class for those weeks. 
+
+## Project
+
+There is a semester long project for the class. It requires teams of three people and is meant to address something currently occurring in data center research. There are three milestones for the project. The first is before spring break, the second midway between spring break and the end of the semester, with the last milestone in the final week of classes.
+The project is not meant to be time intensive, however it does contribute the most to your final course grade. The professors and TAs are happy to help you come up with an interesting topic. 
+Your team will have to write a report for each milestone to show progress.
```
Comment (mikinty) [inline electives/18847.md]: How long are these reports? Are they graded harshly?

File: electives/18847.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+- I greatly enjoyed being exposed to new topics each week.
+- Do not have to be familiar with every topic covered.
+- Class discussions are very interesting and you learn a lot during them.
+- TAs and professors are very approachable and easy to contact, since the class has a slack workspace.
+
+## Cons of the Class
+
+- Fast paced, difficult to ingest new information so frequently.
+- Not enough time is given to the project in class.
+- Reviews often take up much time in the week.
```
Comment (mikinty) [inline electives/18847.md]:
```suggestion
- Reviews often take up a lot of time in the week. 
```

File: electives/18847.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+There is a semester long project for the class. It requires teams of three people and is meant to address something currently occurring in data center research. There are three milestones for the project. The first is before spring break, the second midway between spring break and the end of the semester, with the last milestone in the final week of classes.
+The project is not meant to be time intensive, however it does contribute the most to your final course grade. The professors and TAs are happy to help you come up with an interesting topic. 
+Your team will have to write a report for each milestone to show progress. 
+
+## Pros of the Class
+
+- I greatly enjoyed being exposed to new topics each week.
+- Do not have to be familiar with every topic covered.
+- Class discussions are very interesting and you learn a lot during them.
+- TAs and professors are very approachable and easy to contact, since the class has a slack workspace.
```
Comment (mikinty) [inline electives/18847.md]:
```suggestion
- TAs and professors are very approachable and easy to contact, since the class has a Slack workspace.
```

**Review by mikinty — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (gfieni2022) [reply README.md]: Moved.

File: electives/18847.md (hunk 4 above)
Comment (gfieni2022) [reply electives/18847.md]: It depends on how familiar you are with the material and how good of a presentation you want to make it.

File: electives/18847.md (hunk 3 above)
Comment (gfieni2022) [reply electives/18847.md]: See commit https://github.com/gfieni2022/CMU-ECE-CS-Guide/commit/c2eddb433ab1014933531ef80057d0447d8dc86c

File: electives/18847.md (hunk 6 above)
Comment (gfieni2022) [reply electives/18847.md]: See commit https://github.com/gfieni2022/CMU-ECE-CS-Guide/commit/c2eddb433ab1014933531ef80057d0447d8dc86c

File: electives/18847.md (hunk 4 above)
Comment (gfieni2022) [reply electives/18847.md]: See commit https://github.com/gfieni2022/CMU-ECE-CS-Guide/commit/c2eddb433ab1014933531ef80057d0447d8dc86c

**Review by mikinty — APPROVED** [review]
Thanks, looks good

**Pull request merged**